pytest_django/plugin.py

```diff
@@ -170,6 +179,14 @@ def pytest_load_initial_conftests(early_config, parser, args):
     else:
         _django_project_scan_outcome = PROJECT_SCAN_DISABLED
 
+    # Configure FAIL_INVALID_TEMPLATE_VARS
+    itv = (options.itv or
+           early_config.getini(INVALID_TEMPLATE_VARS_ENV) or
+           os.environ.get(INVALID_TEMPLATE_VARS_ENV))
```

Comment:
This should be changed to prefer the environment variable before the config, see https://github.com/pytest-dev/pytest-django/pull/199.